src/libtcod/bresenham.h

```diff
@@ -95,11 +96,14 @@ namespace tcod {
     This class is provisional.
  */
 class TCODLIB_API BresenhamLine {
+  using cell = std::pair<int, int>;
```

Comment:
I'd like to use `std::array<int, 2>` and then index the variable with `cur_.at(n)` for now.  The typename should describe that this is a 2 item position.

Reply:
Would something like `xy_pos` work for this?

`using xy_pos = std::array<int, 2>;`

Reply:
That would work.  I tend to name custom typenames as if they're classes.
```suggestion
  using Point2 = std::array<int, 2>;
```